src/utils/validateIdentifierInputs.ts

```diff
@@ -17,8 +17,17 @@ const isUniqueList = (list: string[]): boolean =>
 export const validateIdentifierInputs = (
   identifierInputs: IdentifierInput[]
 ): boolean => {
-  for (const { value, validationRegex, textInputType } of identifierInputs) {
+  for (const {
+    label,
+    value,
+    validationRegex,
+    textInputType,
+  } of identifierInputs) {
     //TODO: switch between different errors based on campaign config
+    // TODO: can set the validation regex here once it's confirmed
+    if (label === "Payment receipt number" && !value) {
```

Comment:
Could this be a good chance to make use of c13n for error messages? No need to do the migration all at once, just for this could be a good start

Reply:
discussed with @andrehadianto on this, further discussion with the FE tech debt team on the approach of c13n for campaign specific error messages will be required before making changes here! so will leave this as is!

Reply:
I see so is the `TODO` on line 26 needed? Since it's part of the tech debt and is already tracked somewhere.

Also what is line 27 `TODO` for?

Reply:
Validation regex is set in SSM policies right?

Reply:
yup! currently default is set to alphanumeric, less than 20, will confirm on this and set in SSM policies.
also removed the TODOs